src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -60,10 +61,21 @@
             <tag name="serializer.normalizer" />
         </service>
 
-        <!-- Resource path naming strategies -->
+        <!-- Resources Operations path resolver -->
+
+        <service id="api_platform.operation_path_resolver" alias="api_platform.operation_path_resolver.router" public="false" />
+
+        <service id="api_platform.operation_path_resolver.router" class="ApiPlatform\Core\Bridge\Symfony\Routing\RouterOperationPathResolver" public="false">
+            <argument type="service" id="api_platform.router" />
+            <argument type="service" id="api_platform.operation_path_resolver.custom" />
+        </service>
+
+        <service id="api_platform.operation_path_resolver.custom" class="ApiPlatform\Core\PathResolver\CustomOperationPathResolver" public="false">
```

Comment:
We need to be clear about the decoration priority, but by not using Symfony DI's service decoration (when in fact it's exactly what you're trying to achieve), the current implementation is clearly inflexible.

> It would be weird to decorate a service defined by the user in case he uses its own path resolver, don't you think ?

I'll address this separately.